region/norfair/west/Ice Beam Gate Room.json

```diff
@@ -395,10 +395,16 @@
       "name": "Careful Kill",
       "requires": [
         "canDodgeWhileShooting",
-        "canTrickyJump"
+        "canTrickyJump",
+        "canPrepareForNextRoom",
+        "canStopOnADime"
       ],
       "clearsObstacles": ["D"],
-      "note": "Avoiding damage from all enemies with just Power Beam is tricky, but doable."
+      "note": [
+        "Avoiding damage from all enemies with just Power Beam is tricky, but doable.",
+        "Enter the room holding angle to remove momentum and avoid getting hit by the Dessgeega."
+      ],
+      "devNote": "FIXME: It is possible to enter with a mockball and avoid all enemies."
```

Comment:
The mockball is for [7, 3].  Entering with a mockball doesn't work as a way to dodge everything.